2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,9 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+ const contents =pantry.fridgeContents.concat(pantry.cupboardContents);
+ const missingIngredients = recipe.ingredients.filter(ingredients => contents.indexOf(ingredients) === -1);
```

Comment:
This works, but using the method .includes() probably makes more semantic sense that .indexOf().